File: examples/java/HelloWorld.java
package timezra.codemime;

public abstract class HelloWorld {
 
	String name;

	public static void main(final String[] args) {
		final HelloWorld helloWorld = new HelloWorld() {
			@Override void sayHello() { System.out.println("Hello, " + name); }
		};
		helloWorld.name = System.getProperty("user.name");
		helloWorld.sayHello();
	}
 
	abstract void sayHello();
}

File: examples/scala/HelloWorld.scala
package timezra.codemime

object HelloWorld {

  /** doc */ /* block */ // inline
  @throws(classOf[Exception])
  def main(args: Array[String]) {
    val greeting = "Hello"
    var name = "World"
    println(s"${greeting}, ${name}!");
    println(<span id="salutation">{greeting}, {name}!</span>);
  }
}
